src/UserAction/ConfirmationExecutor.php

```diff
@@ -61,17 +61,6 @@ protected function init(): void
      */
     public function afterActionInit(Model\UserAction $action)
     {
-        $getTableName = function ($arr) {
```

Comment:
why it was needed/is not needed now?

Reply:
Was related to previous callbacks issue with reloading and the fact that the executor was added to $app->html.
Should not be needed now.

Reply:
perfect! 👍